zyngui/zynthian_gui_engine.py

```diff
@@ -43,26 +43,23 @@
 # Zynthian Engine Selection GUI Class
 #------------------------------------------------------------------------------
 
-def initializator(cls):
-	cls.init_engine_info()
-	return cls
-
-@initializator
 class zynthian_gui_engine(zynthian_gui_selector):
 
 	single_layer_engines = ["BF", "MD", "PT", "PD", "AE", "CS"]
 	check_channels_engines = ["AE"]
 
-	@classmethod
-	def init_engine_info(cls):
+	def init_engine_info(self):
```

Comment:
My fried, this *MUST* be a class method. Webconf call this function for getting the engine list without creating an instance ...

Reply:
Indeed. Turns out it's not only webconf, just loading a preset also doesn't work this way; I overlooked that while testing. I'll keep the class-level structure and see if we can do smarter filtering on instance level.